text/0008--individual-agency-company-pages.md

```diff
@@ -0,0 +1,131 @@
+- Start Date: 2018-07-24
+- RFC PR: 
+- Gatsby Issue: 
+
+# Summary
+
+As a follow-up to the [Site Showcase](https://next.gatsbyjs.org/showcase/), we want to create individual & organization pages highlighting the individual community members and what they've built. 
+
+We also want to create a free Job Board highlighting opportunities for employment building Gatsby websites at agencies and companies using Gatsby!
+
+# Motivation
+
+Some goals for the different components include:
+
+#### Individuals Pages
+* Highlight contributions made by community members (plugins built, code PRs)
+* "Official" landing pages to help community members in their personal website marketing efforts
+* Allow companies interested in building Gatsby sites to find individuals nearby them who are open for Gatsby projects
+
+#### Agency Pages
+* Highlight agencies building websites on Gatsby
+* "Official" landing pages to help agencies in their marketing efforts
+* Allow companies interested in building Gatsby sites to find agencies nearby them
+* Help agencies hire for Gatsby-related positions by surfacing their postings to the community; give the community a place to look for Gatsby-related jobs.
+
+#### Company Pages
+* Display list of companies using Gatsby to demonstrate it's "enterprise-ready" and fulfills whatever use-case they're looking for (e-commerce, authentication, etc). 
+* Help companies hire for Gatsby-related positions by surfacing their postings to the community; give the community a place to look for Gatsby-related jobs.
+
+# Detailed design
+
+## Pages
+
+#### All people / agencies / companies
+
+One template with multiple URLs, each of which apply different filters:
+  * /people/ (all people)
+  * /agencies/ (all agencies)
+  * /companies/ (all companies)
```

Comment:
I would vote for containing this under a parent slug, e.g. `/community`, to avoid confusion about what this means. Is `https://gatsbyjs.org/companies` a portfolio of Gatsby-owned companies? A list of companies we've worked with? Something else?

`https://gatsbyjs.org/community/companies` disambiguates this: it signifies "these are companies in the Gatsby community".

Other potential parent slugs:

- `/creators` — could be a great way to build an aspirational identity for Gatsby community members to align with: "I'm a Gatsby Creator!"
- `/ecosystem`
- `/showcase` — would require thinking through how this fits into the IA of the sites/starters showcases, but I can see several good arguments for why this might make sense

Reply:
This would be the first three-part slug, but the clarification is really helpful & provides context. I'm 👍 on `/community/companies/`, will edit.